### PR TITLE
feature/TECH 400 migrate to black formatter

### PR DESCRIPTION
**Pull request by SamTheisens**
This PR attempts to finish https://github.com/Vandebron/mpyl/pull/106
It branches from https://github.com/Vandebron/mpyl/pull/149, so best to merge that one first.

 - [x] Integrates format check into linting process
 - [x] Adds black[d] dependency to pipenv, which should set people up for using the `black` plugin in IntelliJ as per the instructions https://black.readthedocs.io/en/stable/integrations/editors.html#as-local-server. After `pipenv install -d`, the plugin was able to set up the `Local instance` automatically.
 - [x] Filter reformat commit from blame
 - [x] Fix some pylint warnings resulting from the reformat

----
### 📕 [TECH-400](https://vandebron.atlassian.net/browse/TECH-400) Investigate (and implement) code formatter danielkoves@vandebron.nl 
Investigate the usage of a formatter tool (sort of like eslint/prettier) to keep our code style like newlines, spacing, alignment etc consistent. [](https://black.readthedocs.io/en/latest/) seems like a promising option.

🏗️ Build [7](https://jenkins.k8s-serv-backend.vdbinfra.nl/job/MPyL%20Pipeline%20-%20Test/job/PR-152/7/display/redirect) ✅ Successful, started by _Sam Theisens_  
🚀 *[cloudfront-service](https://cloudfront-service-152.test.nl/)*, *enrichChargeSessionsJob*, *job*, *kong-sync*, *[nodeservice](https://nodeservice-152.test.nl/)*, *[sbtservice](https://sbtservice-152.test.nl/)*  

[TECH-400]: https://vandebron.atlassian.net/browse/TECH-400?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Jorg88** [comment]
Instead of me scrolling through the entire pr, I believe the only blocker @koeves mentioned is pylint. That seems to be fixed now, so we can actually push this forward Daniel or was there something else?

**Comment by koeves** [comment]
> Instead of me scrolling through the entire pr, I believe the only blocker @koeves mentioned was pylint. That seems to be fixed now, so we can actually push this forward Daniel or was there something else?

No that was it, this PR should have everything I believe

**Review by koeves — APPROVED** [review]

**Review by Jorg88 — APPROVED** [review]

**Comment by SamTheisens** [comment]
https://github.com/Vandebron/mpyl/pull/152#discussion_r1245374830

@Jorg88, that might well be the case. I'm not a repo admin either, maybe @koeves is?
I think it would make sense to set `Deploy` as job for merge requirement, since it depends on the success of all other jobs.

I think you better merge https://github.com/Vandebron/mpyl/pull/149 first. Since this is PR is a follow-up

File: .github/workflows/build-package.yml [hunk 1]
```diff
@@ -5,91 +5,158 @@ on:
     branches-ignore: [ 'main' ]
 
 jobs:
-  Build_Package:
```
[the remark this answers is not in the document]
Comment (SamTheisens) [reply .github/workflows/build-package.yml:8]: I reverted the change here: https://github.com/Vandebron/mpyl/pull/149. Or rather, replaced `Deploy` with `Build_Package`. Which isn't wrong as a name I guess.

[the remark this answers is not in the document]
Comment (SamTheisens) [reply .github/workflows/build-package.yml:8]: @jorg88, @koeves, the PR ist still not mergeable, but I can't see why. Perhaps you need to reselect the build action in the master branch merge rules

[the remark this answers is not in the document]
Comment (Jorg88) [reply .github/workflows/build-package.yml:8]: It just needed a re-run of the build after you rebased, pr is mergable now :)

[the remark this answers is not in the document]
Comment (SamTheisens) [reply .github/workflows/build-package.yml:8]: @Jorg88 by "re-run of the build" you mean a re-run of the Jenkins build? I'll leave the merging to you folks.

[the remark this answers is not in the document]
Comment (Jorg88) [reply .github/workflows/build-package.yml:8]: Yes, re-run of the jenkins mpyl pr pipeline build 😅

[the remark this answers is not in the document]
Comment (SamTheisens) [reply .github/workflows/build-package.yml:8]: Aight, that seems to have done the trick for https://github.com/Vandebron/mpyl/pull/149 too.

**Pull request merged**